packages/shared/src/util/get-error-message/get-error-message.spec.ts

```diff
@@ -0,0 +1,18 @@
+import { getErrorMessage } from './get-error-message';
+
+describe('get-error-message', () => {
+    it('should return error message', () => {
+        expect.hasAssertions();
+        expect(getErrorMessage(new Error('text'))).toStrictEqual('text');
+    });
+
+    it('should return fallback message if first argument is not instance of error', () => {
+        expect.hasAssertions();
+        expect(getErrorMessage({ data: 'text' }, 'fallbackMessage')).toStrictEqual('fallbackMessage');
+    });
```

Comment:
Maybe also add test case for `getErrorMessage({ message: 'text' }, 'fallbackMessage'` to show that `message` will be ignored?